CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # bedrock ChangeLog
 
+## 4.2.1 - 2021-xx-xx
+
+## Changed
+- Replace 'master' language in process, config, and logging to use 'primary'.
```

Comment:
This should note that the older style is still supported but deprecated and will be removed in 5.x.  That will help make it clear this isn't a breaking change now and will help managing releases for the likely situation when we release 5.x and forget about those TODOs in the code.

Reply:
Fixed in 4c4faca2c567dc459e4f208e3f111491025612a8.